schemas/core/leg.json

```diff
@@ -12,14 +12,6 @@
         },
         "signature": {
           "$ref": "components/common.json#/definitions/signature"
-        },
-        "tspProduct": {
```

Comment:
These 2 properties are only used for legCore. legWait and legTransfer obviously does not need product (duh lol)